src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/ProjectFileSynchronizerTest.cs

```diff
@@ -19,21 +19,21 @@ public ProjectFileSynchronizerTest(ITestOutputHelper testOutput)
     {
     }
 
-    [Fact]
-    public void ProjectFileChanged_Added_AddsProject()
-    {
-        // Arrange
-        var projectPath = "/path/to/project.csproj";
-        var projectService = new Mock<RazorProjectService>(MockBehavior.Strict);
-        projectService.Setup(service => service.AddProject(projectPath, null)).Verifiable();
-        var synchronizer = new ProjectFileSynchronizer(LegacyDispatcher, projectService.Object);
-
-        // Act
-        synchronizer.ProjectFileChanged(projectPath, RazorFileChangeKind.Added);
-
-        // Assert
-        projectService.VerifyAll();
-    }
+    //[Fact]
+    //public void ProjectFileChanged_Added_AddsProject()
```

Comment:
Reviewing commit by commit, this just gets removed right? Because of the change to no longer respond to add? (Unless I'm messing up where in the Project* classes I'm thinking)

Reply:
I don't actually remember, but will check. I suspect this is still here, much like the TODO, but can fix it up.